core/module/used_refinements_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../../spec_helper'
+
+describe "Module.used_refinements" do
+  ruby_version_is "3.2" do
+    it "returns list of all refined modules used in current scope" do
+      refined_integer = nil
+      refined_string = nil
+      refined_items = []
```

Comment:
**minor:** Refined module/class - it's `Integer` or `String` classes in this test. They are actually _refined_. So it's more correct to use the term _refinement_ in the test case